### PR TITLE
Fix Focus Bug

### PR DESCRIPTION
**Pull request by rgoessl**
Mouseout event will no longer hide tooltip if element still has focus.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -123,15 +123,15 @@
 		<span class="box" id="right-tap-toggle" tabindex="0">Tap me (right)</span>
 		<d2l-tooltip id="tap-toggle-tooltip" position="right" for="right-tap-toggle" tap-toggle="">Right Tap Toggle</d2l-tooltip>
 		<br>
-		<button id="aria-button">Focus me for aria-describedby</button>
+		<button id="aria-button" tabindex="0">Focus me for aria-describedby</button>
```
Comment (dlockhart) [inline demo/index.html]: These buttons shouldn't require a `tabindex` -- they're focusable by default natively.

Comment (rgoessl) [reply demo/index.html]: changed

**Comment by mpharoah-d2l** [comment]
> A mechanism is available to dismiss the additional content without moving pointer hover
Will anyone actually follow this? Like the native browser tooltip doesn't even have this AFAIK. This seems like a really bizarre requirement.

**Comment by dlockhart** [comment]
It's a complicated one for sure. The intent is for folks with low vision who often operate in a very zoomed-in manor. Their entire viewport may be zoomed in on just a few UI elements, and they rely on moving the mouse a lot to pan around. Having tooltips appear and block stuff makes it really difficult for them. Similar for users who are zoomed and use the keyboard to move around.

The solution here is to simply dismiss the tooltip immediately if they process escape.

**Comment by dlockhart** [comment]
To answer your question about native tooltips:
1. They don't appear on-focus
2. They can be entirely disabled at the operating system level

**Comment by MichaelKovalchuk** [comment]
Do we know why the tests are failing?

**Pull request merged**